src/phoenix/experimental/evals/models/openai.py

```diff
@@ -175,23 +306,28 @@ def _generate_with_retry(self, **kwargs: Any) -> Any:
             self._openai_error.ServiceUnavailableError,
         ]
 
+        async def metered_openai_completion(**kwargs: Any) -> Any:
+            limit = self.rate_limiter.alimit(self.model_name, openai_token_cost)
+            response = await limit(self._openai.ChatCompletion.acreate)(**kwargs)
+            return response
```

Comment:
Suggestion: Use a name that conveys that the method is asynchronous.

Reply:
I know this is common, but when we aren't pairing a synchronous and asynchronous version I don't think it's necessary: 'async def async_...' feels redundant